### ISSUE TITLE
when training will complete

### ISSUE THREAD

**Issue by huahandsome** [tensorflow/models#2844]
Hello,

When I tried to tune a inception model to identify the flowers, I found that the program never end. 

I read and follow the step posted in the link: https://github.com/tensorflow/models/tree/master/research/inception

The command I used to run the training is:
/usr/bin/python /share_folder/220Proj/flower_demo/bazel-bin/inception/flowers_train.runfiles/inception/inception/flowers_train.py \
--train_dir=/share_folder/220Proj/flower_demo/flowers_train \
--data_dir=/share_folder/220Proj/flower_demo/flowers_data \
--pretrained_model_checkpoint_path=/usr/local/lib/python3.5/dist-packages/inception-v3/model.ckpt-157585 \
--fine_tune=True \
--initial_learning=0.001 \
--input_queue_memory_factor=1

Here is part of the log:
2017-11-21 01:23:16.037863: step 250070, loss = 0.98 (77.4 examples/sec; 0.413 sec/batch)
2017-11-21 01:23:20.213151: step 250080, loss = 0.91 (76.4 examples/sec; 0.419 sec/batch)
2017-11-21 01:23:24.387518: step 250090, loss = 0.95 (77.1 examples/sec; 0.415 sec/batch)
2017-11-21 01:23:28.536253: step 250100, loss = 1.12 (76.9 examples/sec; 0.416 sec/batch)
2017-11-21 01:23:34.435414: step 250110, loss = 0.92 (76.5 examples/sec; 0.418 sec/batch)
2017-11-21 01:23:38.599515: step 250120, loss = 1.07 (77.5 examples/sec; 0.413 sec/batch)
2017-11-21 01:23:42.764535: step 250130, loss = 0.95 (77.0 examples/sec; 0.415 sec/batch)
2017-11-21 01:23:46.922841: step 250140, loss = 0.92 (78.0 examples/sec; 0.410 sec/batch)
2017-11-21 01:23:51.092711: step 250150, loss = 0.96 (77.4 examples/sec; 0.414 sec/batch)

You can see the loss is around 0.98 to 1.07, and never decrease. 

So could you tell if there is max steps in inception or not?

What should I do next? End the program manually?

Thanks,

Shuai Hua

**Comment by aselle** [comment]
Yes, you end it manually. You can then check if your model is sufficiently accurate.  Usually researchers use tensorboard to monitor the loss graph and when it plateaus they call it done. The loss is not a direct measure of accuracy, you need to see what percentage of a held out testing set is correctly classified. Good Luck!
